automatic/notepadplusplus.install/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,30 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageName = 'notepadplusplus'
+$url32       = ''
+$url64       = ''
+$checksum32  = ''
+$checksum64  = ''
+
+$packageArgs = @{
+  packageName            = $packageName
+  fileType               = 'EXE'
+  url                    = $url32
+  url64bit               = $url64
+  checksum               = $checksum32
+  checksum64             = $checksum64
+  checksumType           = 'sha256'
+  checksumType64         = 'sha256'
+  silentArgs             = '/S'
+  validExitCodes         = @(0)
+  registryUninstallerKey = 'notepad\+\+'
+}

```

Comment:
Almost 0 isnt 0.

It must be se this way when u want to set url32 equal to url64 and the same for checksums.

Here it isnt mandatory but i dont like to change the template and regex just for that.

Plus,  the intene is clear here .... above lines are changed by the updater, bellow lines are not.